internal/util/util.go

```diff
@@ -35,6 +35,7 @@ func New(config *configs.Root, configFile string, observation *observ.Observatio
 	if err != nil {
 		log.Fatalf("Failed to initialize AWS Session: %s", err)
 	}
+	awsSession.Config.Region = config.Region
```

Comment:
Yeah, it might not be configured the regions from the credentials file.

There's actually a few other places, though, where we set this default to `us-east-1`. I'd rather this isn't scattered all over the code base. Can we get it centralized in a single spot somewhere?

eg
https://github.com/Optum/dce-cli/blob/9757e4a8194315778cc0b14b6eb4b44455f5679d/pkg/service/init.go#L18

https://github.com/Optum/dce-cli/blob/9757e4a8194315778cc0b14b6eb4b44455f5679d/internal/util/api.go#L31